CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Minor: Delete all but the last 5 crashdumps on application start. (#4392)
 - Minor: Added `/banid` command that allows banning by user ID. (#4411)
+- Minor: FrankerFazeZ animated emotes and authors for global emotes are now shown. (#4434)
```

Comment:
Something like "Added support for FrankerFaceZ animated emotes.", and if we want a changelog entry for the author then just put that as a separate changelog entry as its value is a lot lower

Reply:
The author addition just comes from the unification of parsing - I don't think it's that important to mention.